03_letters/srini/main_test.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var testMatrix = []struct {
+	input      string
+	lettersMap map[rune]int
+	sortedKeys []string
+}{
+	{"", map[rune]int{}, []string{}},
+
+	{"abba",
+		map[rune]int{rune('a'): 2, rune('b'): 2},
+		[]string{"a:2", "b:2"}},
+
+	{"abb a",
+		map[rune]int{rune(' '): 1, rune('a'): 2, rune('b'): 2},
+		[]string{" :1", "a:2", "b:2"}},
+
+	{"£€€€§‡®😎",
+		map[rune]int{rune('£'): 1, rune('‡'): 1, rune('®'): 1, rune('€'): 3, rune('§'): 1, rune('😎'): 1},
```

Comment:
Remove all the `rune(...)` casts in the whole table, replacing with just `...`.

Reply:
fixed this